graphql_compiler/tests/test_helpers.py

```diff
@@ -407,3 +435,13 @@ def get_test_macro_registry():
     for graphql, args in valid_macros:
         register_macro_edge(macro_registry, graphql, args)
     return macro_registry
+
+
+def ordered(obj):
+    """Return an ordered representation of a json-serializable object"""
+    if isinstance(obj, dict):
```

Comment:
Why do we need dicts to be ordered? Dicts (not `OrderedDict`, just vanilla ones) should compare equal if they contain the same keys and values, regardless of order. I think this part is probably not necessary.